packages/form-js-editor/src/features/properties-panel/entries/AltTextEntry.js

```diff
@@ -0,0 +1,60 @@
+import { get } from 'min-dash';
```

Comment:
I think this is an oversight: If the `source` can be defined through input data, then we also need a mechanism to define `altText` through input data, because the whole point of using input data is the flexibility of not knowing what the image will be. From an a11y standpoint I don't think we can make due without it.

Reply:
Good point. Maybe let's follow the same direction here:
* alt: static value (and potentially FEEL later)
* altRef: key to bind input data

Reply:
It makes sense to me, and we could even get rid of altRef in the future if we figure out a safe mechanism to do migrations. Then it would be expressed as a feel expression instead, same as with source.